src/ui/RecipeForm.tsx

```diff
@@ -0,0 +1,115 @@
+import { FC } from 'react';
+import { Formik, Field } from 'formik';
+import Input from './Input';
+import Select from './Select';
+import IngredientForm from './IngredientForm';
+import InputPrepTime from './InputPrepTime';
+import DirectionsBox from './DirectionsBox';
+import TextArea from './TextArea';
+import Button from './Button';
+const bydels = [
+  'Alna',
+  'Bjerke',
+  'Frogner',
+  'Gamle Oslo',
+  'Grorud',
+  'Grünerløkka',
+  'Nordre Aker',
+  'Nordstrand',
+  'Sagene',
+  'St. Hanshaugen',
+  'Stovner',
+  'Søndre Nordstrand',
+  'Ullern',
+  'Vestre Aker',
+  'Østensjø',
+];
+const maxDropdown = 12;
+
+const categories = ['Appetizer', 'Entree', 'Drink', 'Other'];
+
+interface RecipeFormProps {
```

Comment:
I think this component will probably only be used once so we probably don't need this className prop since it could just live directly in this file. and typically we won't be using classNames since we prefer the styled components syntax

Reply:
My understanding is that emotion only lets you style custom components that take a className prop. <Formik> doesn't, hence the wrapper. The form gets styled with emotion on the YourRecipe page. If there's a better way to do this, let me know.

Reply:
nvm, figured it out